Sources/TuistAutomation/Simulator/SimulatorController.swift

```diff
@@ -100,4 +106,12 @@ final class SimulatorController: SimulatorControlling {
                 }
             }
     }
+
+    public func bootSimulator(_ simulator: SimulatorDevice) throws {
+        logger.log(level: .notice, "Booting \(simulator.description)", metadata: .section)
+        _ = try System.shared.observable(["/usr/bin/xcrun", "simctl", "boot", "\(simulator.udid)"])
+            .mapToString()
+            .toBlocking()
```

Comment:
@itsmathou what about returning a `Completble` from this method instead of making it synchronous? That way we can easily combine multiple asynchronous actions for managing the simulator, and run them in parallel to the compilation of the app.